test/WebJobs.Script.Tests.Integration/SamplesEndToEndRestartTests.cs

```diff
@@ -1,79 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Net.Http;
-using System.Threading.Tasks;
-using Newtonsoft.Json.Linq;
-using Xunit;
-
-namespace Microsoft.Azure.WebJobs.Script.Tests
-{
-    // Running these in their own class with private fixtures so they do not affect the timing of
-    // subsequent tests.
-    public class SamplesEndToEndRestartTests
-    {
-        [Fact(Skip = "Will not pass until https://github.com/Azure/azure-webjobs-sdk-script/issues/1537 is fixed")]
```

Comment:
Why not move it to the E2E tests? We don't really care how long those run. The reality is that we have some complicated interactions here that we wouldn't have caught without an overall test like this. I'd hate to just remove it.